src/main/java/org/commcare/core/process/CommCareInstanceInitializer.java

```diff
@@ -168,7 +168,7 @@ protected AbstractTreeElement setupSessionData(ExternalDataInstance instance) {
         if (this.mPlatform == null) {
             throw new RuntimeException("Cannot generate session instance with undeclared platform!");
         }
-        User u = mSandbox.getLoggedInUser();
+        User u = mSandbox.getLoggedInUserUnsafe();
```

Comment:
In the call trace on that Crashlytics crash, the SUE will get caught by the `SessionAwareCommCareActivity` infrastructure. But I'm not 100% sure that all paths to `setupSessionData()` come from somewhere that originated with a `SessionAwareCommCareActivity`, so in some cases it may crash.